libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -194,7 +195,19 @@ def _convert_to_prompt(part: Union[str, Dict]) -> Part:
             path = part["image_url"]["url"]
             return ImageBytesLoader(project=project).load_gapic_part(path)
 
-        raise ValueError("Only text and image_url types are supported!")
+        # Handle media type like LangChain.js
+        # https://github.com/langchain-ai/langchainjs/blob/e536593e2585f1dd7b0afc187de4d07cb40689ba/libs/langchain-google-common/src/utils/gemini.ts#L93-L106
+        if part["type"] == "media":
+            if "mime_type" not in part:
```

Comment:
I went with snake_case for mime_type and file_uri contrary to what's happening in langchain.js as it seemed more pythonic. Do you think that's the right call?

Reply:
can we construct the proto Part directly? why do we need a VertexPart?
and it's probably a good idea to allow to pass gcs_uri as file_data and video_metadata too.

Reply:
>can we construct the proto Part directly?

Done.

>video_metadata

Done.

>pass gcs_uri

file_uri expects a GCS uri, e.g. "gs://bucket/some_video.mp4". Do you mean we should also support http uri-s here?

Reply:
so there's no way to send the video into a base64 uri?, instead of an GCS uri?